core/src/main/java/com/linecorp/armeria/internal/server/JacksonUtil.java

```diff
@@ -37,6 +37,12 @@
 
     private static boolean noticed;
 
+    private static final ObjectMapper INSTANCE = newDefaultObjectMapper();
+
+    public static ObjectMapper getDefaultObjectMapper() {
+        return INSTANCE;
+    }
```

Comment:
How about adding `writeValueAsBytes()` in this class instead of this method, so that we never return a shared `ObjectMapper`?

Reply:
That sounds good.